do.py

```diff
@@ -891,6 +893,23 @@ def perf_script_ldlat(fields, tag): return perf_script("-F %s | grep -v mem-load
     exe(' '.join((perf, 'script', x)), msg=None, redir_out=None)
     print('firefox %s.svg &' % perf_data)
 
+  if en(21):
+    depths=pmu.cpu_pipeline_width('all_depths')
+    evts=""
+    for i in depths:
+      for j in range(int(depths[i][1])):
+        evt_name=depths[i][0]
+        evt_cmask=str(j+1)
+        evts+=str(pmu.perf_event(evt_name+':c'+evt_cmask)).replace(evt_name,evt_name+':c'+evt_cmask)+','
+    evts=evts[:-1]
+    if do['interval'] < 1000:
+      adjusted_interval=int(do['interval']*int((1000/do['interval']))+1000%do['interval'])
```

Comment:
I think this calculation always yields 1000 for any value of do['interval'] < 1000, if so you can just assign 1000 to it.

Reply:
so you can't be less than 1000ms, but you can be greater say 2000ms.